### DISCUSSION TITLE
1.19.2 port.

### DISCUSSION THREAD

**Discussion by murao-kun** [ThePandaOliver/Pandas-Extra-Details#7] [Ideas]
This version is way popular now. Would be cool to have support for that.

**Comment by Eduardobraga13** [comment]
please make it for minecraft version 1.19.2 please

**Comment by ThePandaOliver** [comment]
Im planning on porting it to 1.19.2 when im sure the mod is working as intended.

**Comment by murao-kun** [comment]
Thank you! :)

**Comment by Eduardobraga13** [comment]
ok muito obrigado pela sua atenção

**Comment by ThePandaOliver** [comment]
Sorry to everyone thinking the mod will come to 1.19.2, it will not, porting it will be to much of a pain to do.
